tfrecorder/schema.py

```diff
@@ -25,56 +25,64 @@
 from tensorflow_transform.tf_metadata import dataset_metadata
 from tensorflow_transform.tf_metadata import schema_utils
 
+# TODO(mikebernico): Refactor types into data classes
 # All supported types will be based on _supported_type.
-_supported_type = collections.namedtuple(
+supported_type = collections.namedtuple(
```

Comment:
I think it would be good to use CamelCase here. It would make transitioning to dataclasses/protos easier.

Reply:
Done